recipes/pash/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
 
 about:
   home: http://www.bioinformatics.bbsrc.ac.uk/projects/bismark/
```

Comment:
should contact author, if license is not specified, it is not sure we ware allowed to redistribute software....